### PR TITLE
br: make inc + log restore compatible

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-incremental-guide.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 
 增量备份尚不支持批量重命名表的操作。如果在增量备份过程中发生了批量重命名表的操作，则有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
 
+在 8.3 版本后，新增配置参数 `--allow-pitr-from-incremental`（默认为 true) 来控制增量备份和后续的日志备份是否兼容。兼容的情况下，增量恢复开始前会对需要回放的 DDL 进行严格检查，该模式目前尚不支持 3 种 DDL，分别是 ADD INDEX/MODIFY COLUMN/REORG PARTITION，因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述类型的 DDL。如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental=false` 来跳过增量恢复阶段的检查。
```
Comment (hfxsd) [inline br/br-incremental-guide.md:20]:
```suggestion
在 v8.3.0 版本中，新增了配置参数 `--allow-pitr-from-incremental`，用于控制增量备份和后续的日志备份是否兼容。默认值为 `true`，即增量备份兼容后续的日志备份。兼容的情况下，增量恢复开始前会对需要回放的 DDL 进行严格检查。该模式目前尚不支持 `ADD INDEX`、`MODIFY COLUMN`、`REORG PARTITION`，因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述 DDL。如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental=false` 来跳过增量恢复阶段的检查。
```

File: br/br-pitr-manual.md [hunk 2]
```diff
@@ -341,6 +341,10 @@ tiup br log metadata –-storage='s3://backup-101/logbackup?access-key=${access-
 
 ## 恢复到指定时间点 PITR
 
+> **注意：**
+>
+> - 如果 restore point 指定 --full-backup-storage 是增量备份地址，那么需要保证该备份，以及之前的任意增量备份的恢复，均设置了 `--allow-pitr-from-incremental=true`。
```
Comment (hfxsd) [inline br/br-pitr-manual.md:346]:
```suggestion
> - 如果 `restore point` 指定 `--full-backup-storage` 是增量备份地址，那么需要保证该备份，以及之前的任意增量备份的恢复，均设置了 `--allow-pitr-from-incremental=true`，使增量备份兼容后续的日志备份。
```

**Review by BornChanger — APPROVED** [review]

File: br/br-incremental-guide.md [hunk 3]
```diff
@@ -17,6 +17,8 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 
 增量备份尚不支持批量重命名表的操作。如果在增量备份过程中发生了批量重命名表的操作，则有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
 
+在 v8.3.0 版本中，新增了配置参数 `--allow-pitr-from-incremental`，用于控制增量备份和后续的日志备份是否兼容。默认值为 `true`，即增量备份兼容后续的日志备份。兼容的情况下，增量恢复开始前会对需要回放的 DDL 进行严格检查。该模式目前尚不支持 `ADD INDEX`、`MODIFY COLUMN`、`REORG PARTITION`，因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述 DDL。如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental=false` 来跳过增量恢复阶段的检查。
```
Comment (hfxsd) [inline br/br-incremental-guide.md:20]:
```suggestion
在 v8.3.0 版本中，新增了配置参数 `--allow-pitr-from-incremental`，用于控制增量备份和后续的日志备份是否兼容。默认值为 `true`，即增量备份兼容后续的日志备份。兼容的情况下，增量恢复开始前会对需要回放的 DDL 进行严格检查。该模式目前尚不支持 `ADD INDEX`、`MODIFY COLUMN`、`REORG PARTITION`，因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述 DDL。如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental` 为 `false` 来跳过增量恢复阶段的检查。
```

File: br/br-pitr-manual.md [hunk 4]
```diff
@@ -341,6 +341,10 @@ tiup br log metadata –-storage='s3://backup-101/logbackup?access-key=${access-
 
 ## 恢复到指定时间点 PITR
 
+> **注意：**
+>
+> - 如果 `restore point` 指定 `--full-backup-storage` 是增量备份地址，那么需要保证该备份，以及之前的任意增量备份的恢复，均设置了 `--allow-pitr-from-incremental=true`，使增量备份兼容后续的日志备份。
```
Comment (hfxsd) [inline br/br-pitr-manual.md:346]:
```suggestion
> 如果 `restore point` 指定 `--full-backup-storage` 为增量备份地址，那么需要保证该备份，以及之前的任意增量备份的恢复，均将参数 `--allow-pitr-from-incremental` 设置为 `true`，使增量备份兼容后续的日志备份。
```

**Review by hfxsd — APPROVED** [review]

File: br/br-incremental-guide.md [hunk 5]
```diff
@@ -17,6 +17,8 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 
 增量备份尚不支持批量重命名表的操作。如果在增量备份过程中发生了批量重命名表的操作，则有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
 
+在 v8.3.0 版本中，新增了配置参数 `--allow-pitr-from-incremental`，用于控制增量备份和后续的日志备份是否兼容。默认值为 `true`，即增量备份兼容后续的日志备份。兼容的情况下，增量恢复开始前会对需要回放的 DDL 进行严格检查。该模式目前尚不支持 `ADD INDEX`、`MODIFY COLUMN`、`REORG PARTITION`，因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述 DDL。如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental` 为 `false` 来跳过增量恢复阶段的检查。
```
Comment (hfxsd) [inline br/br-incremental-guide.md:20]:
```suggestion
在 v8.3.0 版本中，新增了配置参数 `--allow-pitr-from-incremental`，用于控制增量备份和后续的日志备份是否兼容。默认值为 `true`，即增量备份兼容后续的日志备份。兼容的情况下，增量恢复开始前会对需要回放的 DDL 进行严格检查。该模式目前尚不支持 `ADD INDEX`、`MODIFY COLUMN`、`REORG PARTITION`，因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述 DDL。否则这三个 DDL 无法被正确 replayed。如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental` 为 `false` 来跳过增量恢复阶段的检查。
```

File: br/br-incremental-guide.md [hunk 6]
```diff
@@ -17,6 +17,8 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 
 增量备份尚不支持批量重命名表的操作。如果在增量备份过程中发生了批量重命名表的操作，则有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
 
+在 v8.3.0 版本中，新增了配置参数 `--allow-pitr-from-incremental`，用于控制增量备份和后续的日志备份是否兼容。默认值为 `true`，即增量备份兼容后续的日志备份。兼容的情况下，增量恢复开始前会对需要回放的 DDL 进行严格检查。该模式目前尚不支持 `ADD INDEX`、`MODIFY COLUMN`、`REORG PARTITION`，因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述 DDL。否则这三个 DDL 无法被正确回放。如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental` 为 `false` 来跳过增量恢复阶段的检查。
```
Comment (lilin90) [inline br/br-incremental-guide.md:20]:
```suggestion
在 v8.3.0 版本中，新增了配置参数 `--allow-pitr-from-incremental`，用于控制增量备份和后续的日志备份是否兼容。默认值为 `true`，即增量备份兼容后续的日志备份。

- 保持默认配置 `true` 时，增量恢复开始前会对需要回放的 DDL 进行严格检查。该模式目前尚不支持 `ADD INDEX`、`MODIFY COLUMN`、`REORG PARTITION`。因此要想搭配使用增量备份和日志备份，需确保增量备份的过程中没有上述 DDL，否则这三个 DDL 无法被正确回放。
- 如果整个恢复流程中只需要增量恢复而不考虑日志备份，则可以通过设置 `--allow-pitr-from-incremental` 为 `false` 来跳过增量恢复阶段的检查。
```

File: br/br-pitr-manual.md [hunk 7]
```diff
@@ -341,6 +341,10 @@ tiup br log metadata –-storage='s3://backup-101/logbackup?access-key=${access-
 
 ## 恢复到指定时间点 PITR
 
+> **注意：**
+>
+> 如果 `restore point` 指定 `--full-backup-storage` 为增量备份地址，那么需要保证该备份、以及之前的任意增量备份的恢复，均将参数 `--allow-pitr-from-incremental` 设置为 `true`，使增量备份兼容后续的日志备份。
```
Comment (lilin90) [inline br/br-pitr-manual.md:346]:
```suggestion
> 如果 `restore point` 指定 `--full-backup-storage` 为增量备份地址，那么需要保证该备份以及之前的任意增量备份的恢复，均将参数 `--allow-pitr-from-incremental` 设置为 `true`，使增量备份兼容后续的日志备份。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**